### PR TITLE
Restaurant list

### PR DESCRIPTION
**Pull request by Saunaheiss**
Dear Manel

Please check and merge.

### CODE REVIEW

File: src/routes/Home/index.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  margin: auto;
+  width: 60%;
+  border: 2px solid rgb(245, 178, 31);
+  background-color: rgba(240, 181, 115, 0.2);
+}
+
+.Home-img {
+  width: 100%;
+  position: fixed;
+}
```
Comment (mapageKA21) [inline src/routes/Home/index.css]: give an empty line

File: src/routes/Home/index.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (mapageKA21) [inline src/routes/Home/index.css]: start coding at line 1!

File: src/routes/Home/index.js [hunk 3]
```diff
@@ -1,18 +1,112 @@
...
+  }
+
+  render() {
+    let restaurants = [];
+    restaurants.push(restaurant1, restaurant2, restaurant3, restaurant4, restaurant5, restaurant6);
+    console.log(restaurants);
+
+    console.log('in render');
+    console.log(restaurants);
+    console.log('end render');
```
Comment (mapageKA21) [inline src/routes/Home/index.js]: Once everything is working, delete the console logs

File: src/routes/Home/index.css (hunk 2 above)
Comment (Saunaheiss) [inline src/routes/Home/index.css]: done

File: src/routes/Home/index.css (hunk 1 above)
Comment (Saunaheiss) [inline src/routes/Home/index.css]: done

File: src/routes/Home/index.js (hunk 3 above)
Comment (Saunaheiss) [inline src/routes/Home/index.js]: done

**Pull request merged**